### ISSUE TITLE
Noevim 0.9 winbar and smooth scrolling

### ISSUE THREAD

**Issue by bfulop** [akiyosi/goneovim#456]
Hello,

I'm using [Lspsaga to display the current document's path](https://github.com/glepnir/lspsaga.nvim#lspsaga-symbols-in-winbar) in the Neovim Winbar.

When using the smooth scrolling, the winbar seems to scroll together with the document, then snap to the top, when the scrolling has finished. The end result is that there seems to be a flickering of the winbar when scrolling the buffer.

Not sure if it's possible to add support for the winbar on the Goneovim side? It would be great if it could be excluded from the scrolling?

Thank you!

**Comment by akiyosi** [comment]
@bfulop 
Hi, Thanks for this report.
I have created a fix to keep winbar rows from scrolling in smooth scrolling.
Any feedback on the behavior would be appreciated.

https://github.com/akiyosi/goneovim/actions/runs/4674981168

**Comment by bfulop** [comment]
Thank you for the quick fixing!

It's working great on my machine. The winbar is no longer jumping with the scrolling but is always fixed to the top of the screen.

**Comment by bfulop** [comment]
Goneovim crashes when I try to accept a suggestion from [Codeium](https://github.com/Exafunction/codeium.vim).

It only happens with this build.

I tried the build from the [latest release](https://github.com/akiyosi/goneovim/actions/runs/4653891162) and it doesn't have this issue.

**Comment by akiyosi** [comment]
@bfulop 
Thanks for confirming.

If possible, could you run goneovim from the command line to reproduce the problem and if you see a stack trace at the time of the crash, could you share it with me?

**Comment by bfulop** [comment]
Here's the log file.

Goneovim closes after this line:

```
2023/04/13 10:26:13.828649 807661.468 finished creating word cache: 'data_fetching_not_yet_im
```

At this point, I pressed TAB to accept the Codeium suggestion.

[codeium-crash.log](https://github.com/akiyosi/goneovim/files/11219979/codeium-crash.log)

There's another way to reproduce the same behaviour (Goneovim auto-closing):

If I have a leftover swap file in (because of a previous crash) `~/.local/state/nvim/swap/%Users%balint.fulop%Desktop%somefile.ts.swp`

and I try to open this file:

If I do:

```
:e ~/Desktop/somefile.ts
```

No issue, Goneovim does not close. ✅ 

If I try to open this file from the Telescope file picker, then Goneovim will close automatically. ❌ 

Here's the log file. Note that I did several attempts to reproduce the issue.

[existing-swap-file-crash.log](https://github.com/akiyosi/goneovim/files/11220109/existing-swap-file-crash.log)
